python/packages/ai/pyproject.toml

```diff
@@ -26,6 +26,8 @@ pytest-asyncio = "^0.21.1"
 black = "^23.7.0"
 isort = "^5.12.0"
 mypy = "^1.5.0"
+httpx = "^0.26.0"
+openai = "^1.11.1"
```

Comment:
is openai supposed to be a dev dependency? Seems like we will need it in the models as a real dep. Also we can find a way to not import httpx since we already have aiohttp as an http client installed.